src/manifest.ts

```diff
@@ -127,27 +132,24 @@ export class Manifest {
         // they can be added and this exception will not be thrown.
         throw e;
       }
+      return;
     }
-    return data;
+    return JSON.parse(content.parsedContent);
   }
 
-  protected async getManifest(): Promise<GitHubFileContents>;
-  protected async getManifest(
-    sha: string
-  ): Promise<GitHubFileContents | undefined>;
-  protected async getManifest(
-    sha?: string
-  ): Promise<GitHubFileContents | undefined> {
+  protected async getManifestJson(): Promise<object>;
```

Comment:
curious why we use the fairly generic `object` type here?

Reply:
kinda lame - I'm pushing an update that locks the types down more strictly.